spec/spec_helper.rb

```diff
@@ -1,7 +1,5 @@
 require "bundler/setup"
 
-ENV['GRAPHQL_RUBY_VERSION'] ||= '1_8'
```

Comment:
As shown above, this can be retrieved from graphql-ruby's constant directly, so probably we no longer need to set this value manually in .travis.yml 🤓